app/models/imports/episode_rss_import.rb

```diff
@@ -0,0 +1,130 @@
+class EpisodeRssImport < EpisodeImport
+  store :config, accessors: [:entry, :audio], coder: JSON
+
+  validates :guid, presence: true
+
+  def set_defaults
+    super
+    self.audio ||= {files: []}
+  end
+
+  def import!
+    status_started!
+    set_audio_metadata!
+
+    status_importing!
+    create_or_update_episode!
+    set_file_resources!
+
+    episode.save!
+    episode.copy_media
+
+    status_complete!
+  rescue => err
+    status_error!
+    raise err
+  end
```

Comment:
Everything below here is copied from `episode_import.rb`.  The code above here was reorganized a bit, but the same logic as it used to have.